src/domains/printit-app/01_network.tf

```diff
@@ -8,6 +8,32 @@ data "azurerm_private_dns_zone" "internal" {
   resource_group_name = local.internal_dns_zone_resource_group_name
 }
 
+data "azurerm_subnet" "apim_vnet" {
+  name                 = local.pagopa_apim_snet
+  resource_group_name  = local.pagopa_vnet_rg
+  virtual_network_name = local.pagopa_vnet_integration
+}
+
+module "printit_pdf_engine_app_service_snet" {
```

Comment:
Please move this code into core-itn, the subnets are created into the core domain